scripts/full_refactoring_analysis.py

```
@@ -260,51 +260,51 @@ def generate_summary(self) -> str:
     os.makedirs(os.path.dirname(summary_file), exist_ok=True)
-    
+
     with open(summary_file, 'w', encoding='utf-8') as f:
         f.write(summary)
-    
+
     print(f"✅ Сводный отчет сохранен: {summary_file}")
-    
+
     # Возвращаем код выхода
     total_checks = sum(1 for r in results.values() if isinstance(r, dict) and "success" in r)
```

Ruff (0.12.2)

Local variable `total_checks` is assigned to but never used

Remove assignment to unused variable `total_checks`

(F841)